src/Parser.js

```diff
@@ -43,8 +43,8 @@ export function parse (value, root, parent, rule, rules, rulesets, pseudo, point
 		switch (previous = character, character = next()) {
 			// (
 			case 40:
-				if (previous != 108 && charat(characters, length - 1) == 58) {
```

Comment:
I have no idea what this `previous` check was doing here. `108` is `l` and I don't see any specific cases related to that. All tests pass too.

Reply:
it's trying to avoid both `:global(` and `url(`

Reply:
would you mind adding a test case for this?

Reply:
```js
test.only('context character VIIII', () => {
    expect(
      stylis(`background: url(i&m&g.png);.a {background: url(i&m&g.png);}`)
    ).to.equal([
      `.user{background:url(i&m&g.png);}`,
      `.user .a{background:url(i&m&g.png);}`
    ].join(''))
  })
```

but we already have a stop gap in 

```js
case 59 + offset: if (ampersand == -1) characters = replace(characters, /\f/g, '')
```

that means that test will pass either way, but if you remove the 

```js
if (ampersand == -1) characters = replace(characters, /\f/g, '')
``` 

you can see it in action, so the check for `l` is an early exit before that.

Reply:
so is it merely an optimization here?

Reply:
Yes

Reply:
ok, i brought this back